### DISCUSSION TITLE
How do I convert these files to wav/mp3?

### DISCUSSION THREAD

**Discussion by NickPlayzGITHUB** [vgmtrans/vgmtrans#342] [Q&A] [answered]
I feel really stupid for asking the question but eh. So I opened up a Kirby mass attack rom and have all the music, the only problem is no matter how hard I've searched on the internet I cannot find how to export these files as Wav or Mp3, as it only gives me the option to export as a Midi, DLS or SF2.
![image](https://user-images.githubusercontent.com/87223282/218616665-944abad5-3cbe-4b94-8412-c733a708acef.png)
can anyone help me?

**Comment by sykhro** [comment, marked as answer]
MP3 is a long string of numbers which can make your speakers vibrate and play sound.

MIDI is like a music score. By itself, it contains no audio. Just information about notes to be played.
DLS and SF2 are like a big folder of WAV files that contain the sounds that instruments make.
These files are useful when you want to edit music. Most people use them with a DAW (Reaper, FL Studio, ...)


If you just want to listen to music, then VGMTrans is not useful to you. You can find all songs you possibly want on websites like ZopharDomain.
Which is a better option because their audio comes from emulators, so it's always accurate.

**Comment by Bonboon229** [comment]
> I feel really stupid for asking the question but eh. So I opened up a Kirby mass attack rom and have all the music, the only problem is no matter how hard I've searched on the internet I cannot find how to export these files as Wav or Mp3, as it only gives me the option to export as a Midi, DLS or SF2. ![image](https://user-images.githubusercontent.com/87223282/218616665-944abad5-3cbe-4b94-8412-c733a708acef.png) can anyone help me?

Literally not the purpose of this program.
Mods, please close this issue.

**Comment by sykhro** [comment]
I like to think that explaining the purpose of the program helped the user :)

**Comment by diogenicnerd** [comment]
@Bonboon229 
> Literally not the purpose of this program.

Oh, the usual geek pedantry. How surprising.
Then if it's not its purpose, then tell me why it's called vgmtrans, that stands for Video Game Music and Trans for... tranforming, from the ROM to the sound, like in VGM. Also explain why we can listen to the musics of the games. And ultimately explain to us why you can export the songs in obscure formats like .sf2 and .dls, and not .wav, which is the most used format because it's lossless? And if we can listen to it, why can't we export it? 
The purpose of t his software is to work with the sound, how comes we don't have the basic feature to work with lossless sound?
> Mods, please close this issue.

Oh yeah, ok, the ultimate nerd. I can picture you pulling your glasses up your nose. I'm so glad I'm not you. I feel sorry for your family.
Anyway guess what, @NickPlayzGITHUB and I are probably not the only ones searching that famous "export in wav" button which sounds like common sense.
